tests/test_frame.py

```diff
@@ -332,25 +332,13 @@ def test_types_mean() -> None:
     df = pd.DataFrame(data={"col1": [2, 1], "col2": [3, 4]})
     s1: pd.Series = df.mean()
     s2: pd.Series = df.mean(axis=0)
-    with pytest.warns(FutureWarning, match="Using the level"):
-        df2: pd.DataFrame = df.mean(level=0)
-    with pytest.warns(FutureWarning, match="Using the level"):
```

Comment:
Given that we are telling people to change from `df.mean(level=0)` to `df.groupby(level=0).mean()`, can you change the old tests to something like that so we make sure that the `groupby` versions are properly typed?